src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -560,6 +562,17 @@ static void read_channel_args(grpc_chttp2_transport* t,
           .value_or(true);
 }
 
+void grpc_chttp2_transport::WriteSecureFrame(grpc_core::SliceBuffer& /*data*/) {
```

Comment:
Going to suggest `WriteSecurityFrame` -- to save confusion for someone consequently assuming that all the other frames are insecure

Reply:
Done.